client/src/main/groovy/de/gesellix/docker/client/stack/DeployConfigReader.groovy

```diff
@@ -761,4 +764,14 @@ class DeployConfigReader {
         log.info("config ${configSpec.keySet()}")
         return configSpec
     }
+
+    List<PlacementPreferences> placementPreferences(List<de.gesellix.docker.compose.types.PlacementPreferences> preferences) {
+        log.info("placementPreferences: ${preferences}")
+        if (preferences == null) {
+            return null
+        }
+        def spread = new Spread(spreadDescriptor: preferences[0].spread)
+        log.info( "spread: ${spread}" )
```

Comment:
Can you adjust the code formatting to be consistent with the surrounding code, please?
```suggestion
        log.info("spread: ${spread}")
```

Reply:
oh yeah, haven't seen it before...